jobs/v3/howto/auto_complete_sample.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2018 Google Inc. All rights reserved.
```

Comment:
We usually use the name `snippets` for the directory. Not a big deal, but nice for consistency.